examples/worlds/actors_population.sdf.erb

```diff
@@ -0,0 +1,190 @@
+<?xml version="1.0" ?>
+<!--
+  This demo shows the actor
```

Comment:
Update description. Can you also add the command used to generate the sdf file here? i.e.

```
erb -T 1 actors_population.sdf.erb > actors_population.sdf
```